choreographer/src/main/java/eu/arrowhead/core/choreographer/service/ChoreographerService.java

```diff
@@ -267,4 +309,22 @@ private UriComponents getOrchestrationProcessUri() {
         }
         throw new ArrowheadException("Choreographer can't find orchestration process URI.");
     }
+
+    public long getRandomElementFromList(List<Long> list) {
+        Random rand = new Random();
+        return list.get(rand.nextInt(list.size()));
+    }
+
+    public OrchestrationFormRequestDTO createOrchestrationFormRequestFromString(String dto) throws IOException {
+        OrchestrationFormRequestDTO orchestrationForm = new ObjectMapper().readValue(dto, OrchestrationFormRequestDTO.class);
```

Comment:
use the same class that you used in store method
then convert to OrchestrationFormRequestDTO